code/modules/atmospherics/machinery/pipes/manifold.dm

```diff
@@ -20,15 +20,18 @@
 		pipe_interference_group = "atmos-[piping_layer]"\
 	)
 
+	var/mutable_appearance/center
+
 /* We use New() instead of Initialize(mapload) because these values are used in update_appearance(UPDATE_ICON)
  * in the mapping subsystem init before Initialize(mapload) is called in the atoms subsystem init.
  * This is true for the other manifolds (the 4 ways and the heat exchanges) too.
  */
-/obj/machinery/atmospherics/pipe/manifold/New()
+/obj/machinery/atmospherics/pipe/manifold/Initialize(mapload)
+	. = ..()
 	icon_state = ""
-	return ..()
+	center = mutable_appearance(icon, "manifold_center")
```

Comment:
You need to add ``center`` in ``update_overlays()`` so it doesn't go away permanently when its appearance is updated